### PR TITLE
Add country to pjnz extract

### PR DESCRIPTION
**Pull request by osymandius**
Add country name to `extract_pjnz_one()`. Presently country name is not extracted when subnational Spectrum files are used.

### CODE REVIEW

**Comment by jeffeaton** [comment]
Thanks very much. 

* Why do you want this?
* If extracting country name, please also extract the ISO3.
* Please add a test

**Comment by r-ash** [comment]
hintr build https://github.com/mrc-ide/hintr/pull/402

**Comment by jeffeaton** [comment]
> Thanks very much.
> 
> * Why do you want this?
> * If extracting country name, please also extract the ISO3.
> * Please add a test

I completed these tasks -- added `iso3` and added a test. I think it is ready to go now.

@osymandius @r-ash -- could you please review and merge?

Thanks,
Jeff

File: R/inputs-spectrum.R [hunk 1]
```diff
@@ -89,17 +89,10 @@ extract_pjnz_one <- function(pjnz) {
...
   spec <- add_shiny90_unaware(spec, pjnz)
 
-
-  spectrum_region_code <- read_spectrum_region_code(pjnz)
-
-  if (spectrum_region_code == 0) {
-    spectrum_region_name <- eppasm::read_country(pjnz)
-  } else {
-    spectrum_region_name <- read_spectrum_region_name(pjnz)
-  }
```
Comment (r-ash) [inline R/inputs-spectrum.R:99]: The hintr test data https://github.com/mrc-ide/hintr/blob/master/tests/testthat/testdata/Malawi2019.PJNZ relied on this conditional to work

```
> read_spectrum_region_name("tests/testthat/testdata/Malawi2019.PJNZ")
[1] NA
> eppasm::read_country("tests/testthat/testdata/Malawi2019.PJNZ")
[1] "Malawi"
```

The NA is causing problems further downstream when trying to generate output. Is this an issue of old test data in hintr or can we still get countries where the spectrum region code is 0?

**Review by r-ash — CHANGES_REQUESTED** [review]
Small issue in hintr

**Review by r-ash — APPROVED** [review]

**Pull request merged**